kotlin-native-tests/src/test/java/EnumsTests.java

```diff
@@ -0,0 +1,42 @@
+
+import com.mirego.interop.java.test.enums.*;
```

Comment:
Isn't it missing the package?

Reply:
Pour les test natif c'est pas nécéssaire, sont tous buildé ensemble.